src/lang/Messages.properties

```diff
@@ -482,6 +482,7 @@ authentication.method.jb.name = JSON-based Authentication
 authentication.method.jb.popup.login.request = {0} : JSON-based Auth Login Request
 authentication.method.pb.field.label.postData				= Login Request POST Data (if any):
 authentication.method.pb.field.label.postDataRequired = Login Request POST Data *:
+authentication.method.pb.field.lable.loginFormUrl			= URL to GET login page(form) *:
```

Comment:
The field is not mandatory, it's possible to leave it empty which leads to issues later when creating the message. Maybe it shouldn't be? If not needed the user can clear it?